west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 3c9414c179f4dcf8500d2c86f1ce6d6a7681afce
+      revision: pull/891/head
```

Comment:
Please update to `624271bddd622a4a0990d60b081da1e137950061`.

Reply:
done

Reply:
Something went wrong. The branch was pushed but without the actual change.

Reply:
Sorry, mistake. Got it right just now.